packages/react-native/ios/BugsnagReactNative/BugsnagReactNative.m

```diff
@@ -56,7 +56,7 @@ @implementation BugsnagReactNative
     self.configSerializer = [BugsnagConfigSerializer new];
 
     if (![Bugsnag bugsnagStarted]) {
-        return nil;
+        [NSException raise:@"BugsnagException" format:@"Failed to initialise Cocoa Client, please check you have added Bugsnag.start() in the application:didFinishLaunchingWithOptions: method of your AppDelegate subclass"];
```

Comment:
This will most likely be Objective-C for a RN project won't it?

```suggestion
        [NSException raise:@"BugsnagException" format:@"Failed to initialise the Bugsnag Cocoa client, please check you have added [Bugsnag start] in the application:didFinishLaunchingWithOptions: method of your AppDelegate subclass"];
```